modules/shmnet.py

```diff
@@ -32,13 +32,14 @@ def forward(self, batch):
             gold_trimaps = batch['trimap'].to(self.args.device)
 
             # Convert to BUF (BG, UNC, FG) classes.
-            gold_trimaps = gold_trimaps.repeat(1, 3, 1, 1)
+            gold_trimaps = gold_trimaps.repeat(1, 3, 1, 1) * 255
```

Comment:
`gold_trimaps` are derived from converting a trimap PNG to grayscale using PIL's `convert` function. 
https://github.com/nikhilweee/semantic-human-matting/blob/a0f1b1ff65cb8fbd1d78e25fc7da7d223ef97cf7/dataset.py#L71 
The [PIL guides](https://pillow.readthedocs.io/en/stable/handbook/concepts.html#concept-modes) suggest that _Each pixel uses the full range of the bit depth. So a 1-bit pixel has a range of 0-1, an 8-bit pixel has a range of 0-255 and so on._ So to the best of my knowledge, I believe that trimaps are already in the range 0-255. Please correct me if I'm wrong.

Reply:
Yes, you are right, but after `F.to_tensor` it will change to 0-1.
https://github.com/pytorch/vision/blob/4d3a309379f2df748922a610c5dd648ae7248bae/torchvision/transforms/functional.py#L138

Reply:
Thank you! I wasn't aware of this fact. Appreciate the help.